src/reporters/console/util.js

```diff
@@ -12,7 +12,8 @@ const CLEAR_RIGHT_OF_CURSOR = 1;
 export function clearLine(stdout: Stdout) {
   if (!supportsColor) {
     if (stdout instanceof tty.WriteStream) {
-      stdout.write(`\r${' '.repeat(stdout.columns - 1)}\r`);
+      const columns = stdout.columns > 0 ? stdout.columns : 100;
```

Comment:
I think rather than being negative, `stdout.columns` equals 0. Then when you make the -1, it becomes negative. Considering that, wouldn't it be better to just update the condition to just be `stdout instanceof tty.WriteStream && stdout.columns > 0`?